x-pack/plugins/cloud_security_posture/server/routes/csp_rule_template/get_csp_rule_template.ts

```diff
@@ -61,13 +60,9 @@ const findCspRuleTemplateHandler = async (
     filter: getBenchmarkTypeFilter(benchmarkId),
   });
 
-  const cspRulesTemplates = await pMap(
-    cspRulesTemplatesSo.saved_objects,
-    async (cspRuleTemplate) => {
-      return { ...cspRuleTemplate.attributes };
-    },
-    { concurrency: 50 }
-  );
+  const cspRulesTemplates = cspRulesTemplatesSo.saved_objects.map((cspRuleTemplate) => {
+    return { ...cspRuleTemplate.attributes };
+  });
```

Comment:
Cloning doesn't seem necessary here
```suggestion
  const cspRulesTemplates = cspRulesTemplatesSo.saved_objects.map((cspRuleTemplate) => cspRuleTemplate.attributes);
```